### PR TITLE
Renaming - support for multiple AudioStreams

### PR DESCRIPTION
**Pull request by thomas-schultz**
Renaming used to chose the audio stream with the highest channel count.
Now it is possible to select the audio stream by index, for example $A1 will be the first AudioStream.

Therefore I have slightly modified the switch statement for token replacement for Movies and TV-series.
It seems to work for me, but I can not guarantee  any side effects, so further testing would be nice.
Especially other regex terms may still not be compatible with this change.

If accepted, this change can also be displayed in the settings screen, otherwise it is kind of a hidden feature.

Thank you!
Thomas

### CODE REVIEW

**Comment by ProIcons** [comment]
First of all you shouldn't work on the Master branch. There's a devel branch that contains the code is about to be merged on master. Master contains some unnecessary commits, that we wouldn't like to have on our devel branch.

To sync your fork with the latest upstream You can follow these instructions:

https://help.github.com/articles/configuring-a-remote-for-a-fork/
https://help.github.com/articles/syncing-a-fork/

In conclusion you can do that to sync your fork.

```
git checkout devel
git remote add upstream https://github.com/tinyMediaManager/tinyMediaManager.git
git fetch upstream
git merge upstream/devel
```

**Comment by thomas-schultz** [comment]
Okay, I have made one commit/push on my fork on the devel branch. Before requesting another Pull-Request, I would like to ask if it is correct this time.

https://github.com/thomas-schultz/tinyMediaManager/commit/b4d3031bbd62854b21bcc96ff95bf028fb58ff0a

**Comment by ProIcons** [comment]
It seems  correct this time. You can check your fork's status on top of it's contents

For instance.

> This branch is 1 commit ahead of tinyMediaManager:devel.

**Comment by thomas-schultz** [comment]
Just to be clear, i have to create another Pull request now?

**Comment by ProIcons** [comment]
Yes, from your Devel branch to our Devel branch.
